csv2docx/cli.py

```diff
@@ -14,6 +14,14 @@
 @click.option(
     '--delimiter', '-d',
     default=";",
-    help='delimiter used in your csv. Default is \';\'')
-def main(data, template, delimiter):
-	c2d.convert(data, template, delimiter)
+    help='Delimiter used in your csv. Default is \';\'')
+@click.option(
+    '--name', '-n',
+    required=True,
```

Comment:
I think making `--name` required is a reasonable approach, though would argue we need to allow flexibility of use. I imagine the scenario of generating un-named tickets, numbered flyers, etc - in which case the name of the files does not need to have a relation to the data. I'll expand on this in a separate issue.

Reply:
Just to add, some columns might be incomplete - which might provide a result the user is not expected. We can have a discussion in the separate issue on what approach might make more sense.

Reply:
Great point about incomplete cells 👍